Support new DELIFEQ command #67

Krish8955 opened the pull request:
- Added DELIFEQ to commands.json
- Regenerated command builders (gen_stream_test.go, gen_string.go, gen_string_test.go)

Signed-off-by: krish8955 <krishnakhandelwal8955@gmail.com>

Krish8955 committed:
Support new DELIFEQ command #67

- Added DELIFEQ to commands.json
- Regenerated command builders (gen_stream_test.go, gen_string.go, gen_string_test.go)

Signed-off-by: krish8955 <krishnakhandelwal8955@gmail.com>

rueian commented on hack/cmds/commands.json:
```diff
@@ -1577,6 +1578,65 @@
+                  }
+              }
+          ],
+          "reply_schema": {
```
We don’t need the command_flags, acl_categories, key_specs, and reply_schema. Could you help remove them? Also please make the indentation consistent.

Krish8955 committed:
Update DELIFEQ command definition

- Removed unnecessary fields (command_flags, acl_categories, key_specs, reply_schema) from DELIFEQ
- Kept only essential metadata for clarity and maintainability
- Kept the indentation maintained

Signed-off-by: Krishna Khandelwal <krishnakhandelwal8955@gmail.com>

rueian commented:
Thank you @Krish8955.

Krish8955 commented:
> Thank you @Krish8955.

You’re welcome! Looking forward to contributing more
